http.go

```diff
@@ -70,6 +70,10 @@ func (s *Server) ServeHTTPS() {
 		log.Fatalf("FATAL: loading tls config (%s, %s) failed - %s", s.Opts.TLSCertFile, s.Opts.TLSKeyFile, err)
 	}
 
+	if len(s.Opts.TLSClientCAFiles) > 0 {
+		config.ClientAuth = tls.RequestClientCert
```

Comment:
also, be aware that requesting a client certificate causes Safari (and IE in some CORS modes) lots of grief. Wondering if we should start a separate port for this.